account.py

```diff
@@ -40,7 +42,8 @@ def multivalue_model(cls, field):
                 'aeat_pending_sii_send', 'aeat_received_sii',
                 'aeat_received_sii_send',
                 'not_allow_out_invoices_aeat_sii_keys',
-                'not_allow_in_invoices_aeat_sii_keys'}:
+                'not_allow_in_invoices_aeat_sii_keys',
+                'sii_default_limit_date'}:
             return pool.get('account.configuration.default_sii')
         return super().multivalue_model(field)
 
```

Comment:
Missing the default to 0.

https://github.com/classmethod
def default_sii_default_offset_days(cls, **pattern):
return 0